packages/admin/src/pages/reviewers/Form/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+	SimpleForm,
+	SelectInput,
+	ReferenceArrayInput,
+	CheckboxGroupInput,
+	ReferenceInput,
+	required,
+} from 'react-admin';
+
+const ReviewersForm = ({ record, save, resource }) => {
+	return (
+		<SimpleForm record={record} save={save} resource={resource}>
+			<ReferenceInput
+				label="User"
+				source="user_id"
+				reference="users"
+				validate={[required()]}
+				perPage={100}
+				fullWidth
+			>
+				<SelectInput optionText="full_name" />
+			</ReferenceInput>
+
+			<ReferenceArrayInput
+				label="Categories"
+				source="categories"
+				reference="terms"
+				validate={[required()]}
+				perPage={100}
+				fullWidth
+				filter={{ taxonomy: 'category' }}
+				format={(v) => {
+					try {
+						return v.map((i) => i.id || i);
```

Comment:
We should keep the code as [clean](https://medium.com/@BrunoLM7/a-brief-summary-of-clean-code-c0c557739551) and clear as possible to make it easier for other developers to maintain it. So, I suggest using meaningful names here, instead of `v` and `i`